src/ecotranslations.jl

```diff
@@ -0,0 +1,13 @@
+# Getters and setters with microbiome-flavored names
+
+# species -> features
+const nfeatures(com::AbstractComMatrix) = nspecies(com)
```

Comment:
I actually mean without the arguments, e.g. literally `const nfeatures = nspecies`,  not sure it matters, though.